docs/streams/developer-guide/datatypes.html

```diff
@@ -69,6 +69,7 @@ <h2>Configuring SerDes<a class="headerlink" href="#configuring-serdes" title="Pe
     </div>
     <div class="section" id="overriding-default-serdes">
       <h2>Overriding default SerDes<a class="headerlink" href="#overriding-default-serdes" title="Permalink to this headline"></a></h2>
+      <p>The auto-configure of user-defined SerDes is done at KafkaStreams construction time.</p>
```

Comment:
Nit: No need to start a new paragraph. `<p>` -- we can put both sentences into one (note, I would like to fix `serde settings` -> `SerDe settings` in the first/existing sentence):
```
<p>You can also specify SerDes explicitly by passing them to the appropriate API methods, which overrides the default SerDe settings.
   For this case, Kafka Streams will auto-configure the passed-in SerDes objects, i.e., you don't need to call <code>configure()</code> manually.</p>